modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/attachments_list.jspf

```diff
@@ -61,4 +61,59 @@
 		markupView="lexicon"
 		paginate="<%= paginate %>"
 	/>
-</liferay-ui:search-container>
\ No newline at end of file
+</liferay-ui:search-container>
+
+<%
+Date expirationDate = new Date(System.currentTimeMillis() + GetterUtil.getInteger(PropsUtil.get(PropsKeys.SESSION_TIMEOUT)) * Time.MINUTE);
+
+Ticket ticket = TicketLocalServiceUtil.addTicket(user.getCompanyId(), User.class.getName(), user.getUserId(), TicketConstants.TYPE_IMPERSONATE, null, expirationDate, new ServiceContext());
+%>
+
+<aui:script use="liferay-search-container">
+	var searchContainer = Liferay.SearchContainer.get('<portlet:namespace />pageAttachments');
+
+	searchContainer.get('contentBox').delegate(
+		'click',
+		function(event) {
+			var A = AUI();
+
+			var link = event.currentTarget;
+
+			var tr = link.ancestor('tr');
+
+			var deleteURL = Liferay.PortletURL.createActionURL();
+
+			var actionName = '/wiki/edit_page_attachment';
+
+			var cmd = '<%= trashHelper.isTrashEnabled(scopeGroupId) ? Constants.MOVE_TO_TRASH : Constants.DELETE %>';
+
+			var userId = <%= user.getUserId() %>;
+
+			var params = {
+				cmd: cmd,
+				nodeId: <%= wikiPage.getNodeId() %>,
+				title: '<%= wikiPage.getTitle() %>',
+				fileName: link.getAttribute('data-fileName'),
+				ticketKey: '<%= ticket.getKey() %>'
+			};
+
+			deleteURL.setDoAsUserId(userId);
+			deleteURL.setName(actionName);
+			deleteURL.setParameters(params);
+			deleteURL.setPortletId('com_liferay_wiki_web_portlet_WikiAdminPortlet');
+
+			A.io.request(
```

Comment:
Hey @robertoDiaz 

I tried to generate the deleteURL by using taglib, the problem is, however, that the fileName parameter can only be acquired as an attribute of the link object. Since the taglibs are already compiled when the page is loaded, the javascript object cannot appended to the generated URL string if the click handler is executed.
The pattern for creating the URL by code I have taken from https://github.com/liferay/liferay-portal/blob/master/modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/remote_services.js#L365